StripePayments/StripePayments/Source/API Bindings/STPAPIClient+Payments.swift

```diff
@@ -887,7 +887,11 @@ extension STPAPIClient {
                 if let paymentMethod = paymentMethod {
                     continuation.resume(with: .success(paymentMethod))
                 } else {
-                    continuation.resume(with: .failure(error ?? NSError.stp_genericFailedToParseResponseError()))
+                    if paymentMethodUpdateParams.card?.networks?.preferred != nil {
+                        continuation.resume(with: .failure(error ?? NSError.stp_cardBrandNotUpdatedError()))
```

Comment:
If we want a specific error about updating the payment method let's just add a generic payment method update failed error.